client/models.py

```diff
@@ -281,7 +284,62 @@ def image_download_url(self):
             return reverse('download_certificate', kwargs={'pk': self.id})
 
 
+class HelpRequestVolunteerManager(models.Manager):
+    def _coord_distance(self, p1, p2):
+        """ Haversine Formula """
+        if USE_NEW_DISTANCE_FORMULA:
+            # Note: This formula does not pass testing yet, returns wrong distances (also unsuitable for sorting).
+            earth_radius = 6371
+            lat1 = math.radians(p1[0])
+            lat2 = math.radians(p2[0])
+            dLat = math.radians(p2[0] - p1[0])
+            dLon = math.radians(p2[1] - p1[1])
+
+            a = math.sin(dLat / 2) ** 2 + math.cos(lat1) * math.cos(lat2) * math.sin(dLon / 2) ** 2
+            c = 2 * math.atan2(math.sqrt(a), math.sqrt(1 - a))
+            return earth_radius * c / 100
+        else:
+            x_diff = (p1[0] - p2[0]) ** 2.0
+            y_diff = (p1[1] - p2[1]) ** 2.0
+            return int(((x_diff + y_diff) ** 0.5) / 100)
+
+    def all_by_distance(self, h_coord):
+        volunteers_qs = Volunteer.objects.all()
+        volunteers_qs = self._add_distance(volunteers_qs, h_coord)
+        # return sorted(volunteers, key=lambda v: v.distance)
+        return volunteers_qs.order_by('distance')
+
+    @staticmethod
```

Comment:
Static methods should be at the top of the class, then private methods, then public methods (the public/private order doesn't matter to me as much as the them being organized)